core-services/prow/02_config/_plugins.yaml

```diff
@@ -634,7 +634,16 @@ config_updater:
         build01:
         - ci
       name: ci-operator-misc-configs
-    ci-operator/config/openshift/kubernetes/*.yaml:
+    ci-operator/config/openshift-priv/kubernetes/openshift-priv-kubernetes*.yaml:
+      clusters:
+        api.ci:
```

Comment:
we don't need ci-operator config on anything but `app,ci`

Reply:
OK, I just copied existing stanzas - I'll make a followup PR with a full cleanup?